pyffi/utils/__init__.py

```diff
@@ -42,7 +42,28 @@
 # ***** END LICENSE BLOCK *****
 
 import os
+from distutils.cmd import Command
 
+class BuildDoc(Command):
```

Comment:
Why would we not have this?

Reply:
That command class is only imported if python can't find sphinx so that pyffi can be installed without sphinx

Reply:
Sorry, probably meant to say when would occur? Presume your thinking is that failing to build docs should not fail the build.

From an automation perspective the danger here is that we don't get notified if something goes wrong. Should probably take the break it fix it approach. Especially if the intent is to upload them.

Reply:
I can make it so that instead of print an message it raises an exception. I don’t really know how setup.py works too well so I thought that if the user doesn’t have Sphinx installed it will raise an exception when trying to install PyFFI because we import the Sphinx BuildDoc in the setup.py

Reply:
Ah, that could be a general problem from either perspective. If a user doesn't want to run docs, they shouldn't need it to install, we should localize scope.

Reply:
Thats why I added [this](https://github.com/niftools/pyffi/pull/46/files/80ae40191766622795d36384fac66c020ac00719#diff-2eeaed663bd0d25b7e608891384b7298R99) to the setup.py so that if people do want to run documentation that just have to `pip install pyffi[docs]` which will also add sphinx to the downloads.

Reply:
Yeah, still alot more thoughts that I would like to run through with you, but  I think at this stage probably best at this stage to take there discussions over on chat as it is cluttering the review. Happy enough to understand why this was done this way and definitely something I would like us to build on. Don't want to hold up the PR any further.